src/components/Navbar.js

```diff
@@ -0,0 +1,41 @@
+//@flow strict
+
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+
+// Navbar component. Displays back button and player's name and currency.
+// The Navbar has 3 columns:
+// The left column is the back button.
+// The middle column is the current page name.
+// The right column is the player's name and current currency.
+//
+// Prop names:
+// styleName (takes the CSS style for the back button, two options are navbtn and nobtn)
+// returnName (takes text for describing the back button)
+// pageName (takes a title for top of the page)
+// userName (takes user's name and currency)
+//
+// EXAMPLE PROP USAGE = <Navbar styleName="navbtn" returnName="Back to Main Menu" pageName="Armory" userName="FRIcker | $465128"/>
+//
+// Right column needs to be updated to have the logged in user's name and current currency passed to it.
+class Navbar extends React.Component<Props> {

```

Comment:
You have to define what the type of Props are. e.g. type Props = {styleName: string, returnName: string, pageName: string, userName: string}

Reply:
Example: https://github.com/RickLeinecker/Fortuna/blob/master/src/casus/blockBank/BlockBank.js

Reply:
I'll fix this soon. First, I need to fix my Flow. Almost forgot about that.

Reply:
Props have been created in both components.